presto-docs/src/main/sphinx/release/release-0.265.rst

```diff
@@ -0,0 +1,25 @@
+=============
+Release 0.265
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Add a configuration property ``fragment-result-cache.max-cache-size`` to control the total on-disk size of fragment result cache. The default value is 100G.
+* Rename function ``array_dupes`` to :func:`array_duplicates`, and rename function ``array_has_dupes`` to :func:`array_has_duplicates`. Previous names are kept as aliases but will be deprecated in the future.
+
+Verifier Changes
+________________
+* Fix verifier executable jar corruption.
```

Comment:
"Fix a verifier ..."

Reply:
What impact does this have ? Can we clarify the change the users would expect ? @ajaygeorge